modules/osdk-monitoring-prometheus-servicemonitor.adoc

```diff
@@ -15,6 +15,6 @@ generate a ServiceMonitor Custom Resource (CR) based on it.
 
 .Additional resources
 
-- See the
-link:https://github.com/coreos/prometheus-operator/blob/7a25bf6b6bb2347dacb235659b73bc210117acc7/Documentation/design.md#servicemonitor[Prometheus Operator documentation]
-for more about the ServiceMonitor CRD.
+* See the
```

Comment:
We are no longer linking to GitHub-hosted content per our [user guide](https://github.com/openshift/openshift-docs/blob/enterprise-4.0/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites)

Reply:
@sferich888 I've heard that the request to not link to GitHub content came from you. Just wanted to run this one by you (since you made an exception in the CLI content) to make sure this is indeed a blanket statement and if this instance should be removed. 

Just want to make sure I understand. Thanks!

Reply:
Why would we not link to downstream docs for that topic?

Reply:
Hi @sferich888, I'm not sure I understand what you're suggesting. It doesn't look like there is more information about ServiceMonitor in that file (which is here: https://docs.openshift.com/container-platform/4.0/monitoring/monitoring.html).

Reply:
After discussing further with @sferich888, I opened https://jira.coreos.com/browse/OSDOCS-394 to cover adding the ServiceMonitor details into the docs, and removing this link altogether. Leaving this link as is for now, and will make that update in a separate PR.